README.md

```diff
@@ -1,3 +1,30 @@
-# jct-discord-bot
+# JCT Discord Bot
 
- Bot for JCT ESP Compsci Discord
+Bot for [JCT ESP Compsci](https://discord.gg/rBsJ48Zz7q) Discord server
```

Comment:
Probably better not to put the invite link since the repo is public and we don't want any random people to join. JCT people who find this already have the link out can ask about it.